src/DotNetWorker.Core/Converters/Converter/InputConverterAttribute.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Functions.Worker.Converters;
+
+namespace Microsoft.Azure.Functions.Worker.Core.Converters.Converter
+{
+    /// <summary>
+    /// An attribute that can specify a type of <see cref="IInputConverter"/> to use for function input conversion.
+    /// </summary>
+    [AttributeUsage(
+        AttributeTargets.Parameter |
+        AttributeTargets.Class |
+        AttributeTargets.Enum |
+        AttributeTargets.Struct)]
+    public sealed class InputConverterAttribute : Attribute
+    {
+        /// <summary>
+        /// Gets the input converter type.
+        /// </summary>
+        public Type ConverterType { get; }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="InputConverterAttribute"/>
+        /// </summary>
+        /// <param name="converterType">The input converter type.</param>
+        /// <exception cref="InvalidOperationException">Thrown when the converterType parameter value is a type which has not implemented Microsoft.Azure.Functions.Worker.Converters.</exception>
+        public InputConverterAttribute(Type converterType)
+        {
+            if (converterType == null)
+            {
+                throw new ArgumentNullException(nameof(converterType));
+            }
+
+            var interfaceType = typeof(IInputConverter);
+            if (interfaceType.IsAssignableFrom(converterType) == false)
```

Comment:
If `IsAssignableFrom` returns a boolean, do we need to add the `== false` part or can we just do `!interfaceType.IsAssignableFrom(converterType)`

Reply:
Compiler will lower `==false` to `!`. I fixed here as well now though.